journalbeat/docs/getting-started.asciidoc

```diff
@@ -0,0 +1,185 @@
+------------------------------------------------
+
+endif::[]
+
+[[rpm]]
+*rpm:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of {beatname_uc} has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes"]
+------------------------------------------------
+curl -L -O https://artifacts.elastic.co/downloads/beats/{beatname_lc}/{beatname_lc}-{version}-x86_64.rpm
+sudo rpm -vi {beatname_lc}-{version}-x86_64.rpm
+------------------------------------------------
+
+endif::[]
+
+[[linux]]
+*linux:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of {beatname_uc} has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes"]
+------------------------------------------------
+curl -L -O https://artifacts.elastic.co/downloads/beats/{beatname_lc}/{beatname_lc}-{version}-linux-x86_64.tar.gz
+tar xzvf {beatname_lc}-{version}-linux-x86_64.tar.gz
+------------------------------------------------
+
+endif::[]
+
+[id="{beatname_lc}-configuration"]
+=== Step 2: Configure {beatname_uc}
+
+Before running {beatname_uc}, you need to specify the location of the systemd
+journal files and configure how you want the files to be read.
+
+include::../../libbeat/docs/shared-configuring.asciidoc[]
+
+Here is a sample of the +{beatname_lc}+ section of the +{beatname_lc}.yml+ file.
+{beatname_uc} uses predefined default values for most configuration options.
+
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+journalbeat.inputs:
+- paths: ["/path/to/journal/directory"]
+  seek: head
+----------------------------------------------------------------------
+
+To configure {beatname_uc}:
+
+. Specify a list of paths to your systemd journal files. Each path can be a
+directory path (to collect events from all journals in a directory), or a file
+path. For example:
++
+["source","sh",subs="attributes"]
+----
+{beatname_lc}.inputs:
+- paths:
+  - "/dev/log"
+  - "/var/log/messages/my-journal-file.journal"
+----
++
+If no paths are specified, {beatname_uc} reads from the default journal.
+
+. Set the <<{beatname_lc}-seek,`seek`>> option to control the position where
+{beatname_uc} starts reading the journal. The available options are `head`,
+`tail`, and `cursor`. The default is `cursor`, which means that
+{beatname_uc} will continue reading where it left off after a reload or restart.
+
+. (Optional) Set the <<{beatname_lc}-include-matches,`include_matches`>> option
+to filter entries in journald before collecting any log events. This reduces the
+number of fields that {beatname_uc} needs to process. For example, to fetch only
+Redis events from a Docker container tagged as `redis`, use: 
++
+["source","sh",subs="attributes"]
+----
+{beatname_lc}.inputs:
+- paths: []
+  include_matches:
+    - "CONTAINER_TAG=redis"
+    - "_COMM=redis"
+----
```

Comment:
Field matching syntax similar to journalctl CLI tool?

Q: why not dictionary syntax?

```
include_matches:
  CONTAINER_TAG: redis
  _COMM: redis
```

Reply:
`journalctl` uses the same functions for matching. However, it has more advanced filters implemented on top of the sdjournal API. For example it has `--units` or `--machine` flags.
Matcher function require this format `{fieldname}={value}`. I do not think it is easier to configure using the dictionary syntax, so I chose this format to save transforming dictionaries to this.

This format is what mheese/journalbeat supports, so users need to change less things their configuration if they migrate to the Elastic version of the beat.

Reply:
@kvch I don't think there are any changes required here, but let me know so I can resolve this item.